client/src/component/TicketsSidebar/index.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react';
+import { BrowserRouter as Router, Link, Route } from 'react-router-dom';
+import './style.css';
+
+class TicketsSidebar extends Component {
+  render() {
+    return (
+      <div className="ticket-sidebar">
+        <button className="ticket-sidebar__new-ticket">
+          <i className="fas fa-plus" />
+          {'          '}
+          Ticket
+        </button>
+        <div className="ticket-sidebar__links-container">
+          <Router>
+            <div className="ticket-sidebar__links ticket-sidebar__selected">
+              <Link to="/tickets/all" className="ticket-sidebar__all-link">
+                All
+                {' '}
+              </Link>
+              <i className="far fa-circle" />
+            </div>
+            <div className="ticket-sidebar__links">
+              <Link to="/tickets/my" className="ticket-sidebar__my-link">My</Link>
```

Comment:
the endpoint /my is kind of a bad naming. It can be my-tickets or something similar